deploy/ansible/feature-deploy.yml

```diff
@@ -52,7 +45,3 @@
         - { drandNumber: 3 }
         - { drandNumber: 4 }
         - { drandNumber: 5 }
-
-- hosts: asset_registries
-  roles:
-    - asset-registry
```

Comment:
Why do we need a separate playbook file for feature deployment? We can reuse the existing `deploy.yml` as is. It doesn't contain the `asset_registries` part and the `goshimmer:&public_nodes` will be empty from the inventory file as it's for `pre-devnet`.

Reply:
For Feature network we also deploy DRNG instances everytime we deploy the network, while in devnet those are created by initial setup and not touched during subsequent deployments. Maybe it could be simplified.

Reply:
Maybe @karimodm can tell you more about it as he created this file.

Reply:
@karimodm If we want drands instances to be deployed at the same time as Goshimmer nodes other environments as `pre-devnet` and `devnet` would benefit from it too. So we add it to the original `deploy.yml` file and we again don't need a separate `feature-deploy.yml`

Reply:
The thing that was added to `feature-deploy.yml` that is not in the original `deploy.yml` is drands.
I agree with you that we don't need to deploy them every time for all our environments including the feature network.
So we can remove the recently added drands part from the `feature-deploy.yml`. That makes it identical with the original `deploy.yml` and we can get rid of the `feature-deploy.yml` at all.